sql/catalog/lease: TestTxnObeysTableModificationTime failed

sql/catalog/lease.TestTxnObeysTableModificationTime [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10839704?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10839704?buildTab=artifacts#/) on release-23.1 @ [15480b8496e7928845330055d191e331b228eb66](https://github.com/cockroachdb/cockroach/commits/15480b8496e7928845330055d191e331b228eb66):


```
I230709 09:55:20.472198 95645 testutils/soon.go:41  [-] 6502  SucceedsSoon: expecting 2 keys, but found 5
I230709 09:55:20.928577 95569 kv/kvserver/queue.go:614  [T1,n1,s1] 6503  rate limited in MaybeAdd (mvccGC): throttled on async limiting semaphore
I230709 09:55:20.929235 95569 kv/kvserver/queue.go:614  [T1,n1,s1] 6504  rate limited in MaybeAdd (consistencyChecker): throttled on async limiting semaphore
I230709 09:55:21.477189 95645 testutils/soon.go:41  [-] 6505  SucceedsSoon: expecting 2 keys, but found 5
I230709 09:55:21.837900 1444146 jobs/registry.go:1588  [T1,n1] 6506  AUTO CREATE STATS job 880889427660898305: stepping through state succeeded
I230709 09:55:22.488329 95645 testutils/soon.go:41  [-] 6507  SucceedsSoon: expecting 2 keys, but found 5
I230709 09:55:22.516007 275685 sql/stats/automatic_stats.go:910  [T1,n1] 6508  automatically executing "CREATE STATISTICS __auto__ FROM [43] WITH OPTIONS THROTTLING 0.9 AS OF SYSTEM TIME '-30s'"
I230709 09:55:23.423418 1466276 jobs/registry.go:1588  [T1,n1] 6509  AUTO CREATE STATS job 880889439378243585: stepping through state running
I230709 09:55:23.502657 95645 testutils/soon.go:41  [-] 6510  SucceedsSoon: expecting 2 keys, but found 5
I230709 09:55:24.523340 95645 testutils/soon.go:41  [-] 6511  SucceedsSoon: expecting 2 keys, but found 5
I230709 09:55:24.558507 95569 kv/kvserver/queue.go:614  [T1,n1,s1] 6512  rate limited in MaybeAdd (replicaGC): throttled on async limiting semaphore
I230709 09:55:24.558872 95569 kv/kvserver/queue.go:614  [T1,n1,s1] 6513  rate limited in MaybeAdd (timeSeriesMaintenance): throttled on async limiting semaphore
I230709 09:55:25.543129 95645 testutils/soon.go:41  [-] 6514  SucceedsSoon: expecting 2 keys, but found 5
I230709 09:55:26.188998 95569 kv/kvserver/queue.go:614  [T1,n1,s1] 6515  rate limited in MaybeAdd (consistencyChecker): throttled on async limiting semaphore
I230709 09:55:26.196916 95569 kv/kvserver/queue.go:614  [T1,n1,s1] 6516  rate limited in MaybeAdd (mvccGC): throttled on async limiting semaphore
I230709 09:55:26.550706 95645 testutils/soon.go:41  [-] 6517  SucceedsSoon: expecting 2 keys, but found 5
I230709 09:55:27.560813 95645 testutils/soon.go:41  [-] 6518  SucceedsSoon: expecting 2 keys, but found 5
I230709 09:55:28.068603 1466276 jobs/registry.go:1588  [T1,n1] 6519  AUTO CREATE STATS job 880889439378243585: stepping through state succeeded
I230709 09:55:28.563482 95645 testutils/soon.go:41  [-] 6520  SucceedsSoon: expecting 2 keys, but found 5
I230709 09:55:28.910269 95910 2@util/log/event_log.go:32  [T1,n1] 6521 ={"Timestamp":1688896528910255753,"EventType":"runtime_stats","MemRSSBytes":1168072704,"GoroutineCount":408,"MemStackSysBytes":6520832,"GoAllocBytes":137689128,"GoTotalBytes":199273768,"GoStatsStaleness":0.005703384,"HeapFragmentBytes":17770456,"HeapReservedBytes":12713984,"HeapReleasedBytes":22372352,"CGoAllocBytes":34867904,"CGoTotalBytes":80269312,"CGoCallRate":0.09997341,"CPUUserPercent":355.00558,"CPUSysPercent":23.693697,"GCPausePercent":0.048332524,"GCRunCount":829,"NetHostRecvBytes":1191,"NetHostSendBytes":1191}
I230709 09:55:28.911687 95910 2@server/status/runtime_log.go:47  [T1,n1] 6522  runtime stats: 1.1 GiB RSS, 408 goroutines (stacks: 6.2 MiB), 131 MiB/190 MiB Go alloc/total(stale) (heap fragmentation: 17 MiB, heap reserved: 12 MiB, heap released: 21 MiB), 33 MiB/77 MiB CGO alloc/total (0.1 CGO/sec), 355.0/23.7 %(u/s)time, 0.0 %gc (829x), 1.2 KiB/1.2 KiB (r/w)net
I230709 09:55:29.564873 95645 testutils/soon.go:41  [-] 6523  SucceedsSoon: expecting 2 keys, but found 5
I230709 09:55:30.567285 95645 testutils/soon.go:41  [-] 6524  SucceedsSoon: expecting 2 keys, but found 5
I230709 09:55:31.569736 95645 testutils/soon.go:41  [-] 6525  SucceedsSoon: expecting 2 keys, but found 5
I230709 09:55:32.571792 95645 testutils/soon.go:41  [-] 6526  SucceedsSoon: expecting 2 keys, but found 5
I230709 09:55:33.574534 95645 testutils/soon.go:41  [-] 6527  SucceedsSoon: expecting 2 keys, but found 5
    lease_test.go:1186: condition failed to evaluate within 3m45s: expecting 2 keys, but found 5
I230709 09:55:34.627657 143971 util/log/event_log.go:32  [T1,n1,client=127.0.0.1:37378,hostssl,user=root] 6528 ={"Timestamp":1688896534576072436,"EventType":"set_cluster_setting","Statement":"SET CLUSTER SETTING \"kv.gc_ttl.strict_enforcement.enabled\" = DEFAULT","Tag":"SET CLUSTER SETTING","User":"root","SettingName":"kv.gc_ttl.strict_enforcement.enabled","Value":"DEFAULT"}
I230709 09:55:34.632086 97083 sql/stats/automatic_stats.go:572  [T1,n1] 6529  quiescing auto stats refresher
I230709 09:55:34.634104 262158 jobs/registry.go:1588  [T1,n1] 6530  AUTO UPDATE SQL ACTIVITY job 103: stepping through state succeeded
I230709 09:55:34.636011 262206 jobs/registry.go:1588  [T1,n1] 6531  KEY VISUALIZER job 100: stepping through state succeeded
W230709 09:55:34.636886 96825 sql/sqlliveness/slinstance/slinstance.go:334  [T1,n1] 6532  exiting heartbeat loop
W230709 09:55:34.637241 96825 sql/sqlliveness/slinstance/slinstance.go:321  [T1,n1] 6533  exiting heartbeat loop with error: node unavailable; try another peer
W230709 09:55:34.637363 1536885 1@sql/event_log.go:659  [T1,n1,client=127.0.0.1:37378,hostssl,user=root] 6534  unable to save 1 entries to system.eventlog: log-event: failed to read query result: query execution canceled
I230709 09:55:34.637882 97084 sql/stats/automatic_stats.go:612  [T1,n1] 6535  quiescing stats garbage collector
E230709 09:55:34.637694 96825 server/server_sql.go:509  [T1,n1] 6536  failed to run update of instance with new session ID: node unavailable; try another peer
I230709 09:55:34.638143 96057 rpc/context.go:2318  [T1,n1,rnode=1,raddr=127.0.0.1:46617,class=default,rpc] 6537  connection heartbeat loop ended with err: <nil>
I230709 09:55:34.638454 95815 server/start_listen.go:103  [T1,n1] 6538  server shutting down: instructing cmux to stop accepting
E230709 09:55:34.639494 262292 jobs/registry.go:989  [T1,n1] 6539  error getting live session: node unavailable; try another peer
I230709 09:55:34.640298 97925 jobs/registry.go:1588  [T1,n1] 6540  AUTO SPAN CONFIG RECONCILIATION job 880888705949171713: stepping through state succeeded
W230709 09:55:34.656062 262158 jobs/adopt.go:518  [T1,n1] 6541  could not clear job claim: clear-job-claim: failed to send RPC: sending to all replicas failed; last error: ba: Scan [/Table/15/1/103,/Table/15/1/104), [txn: fbb73662], [can-forward-ts] RPC error: node unavailable; try another peer
W230709 09:55:34.656819 262206 jobs/adopt.go:518  [T1,n1] 6542  could not clear job claim: clear-job-claim: node unavailable; try another peer
W230709 09:55:34.658252 143001 jobs/adopt.go:518  [T1,n1] 6543  could not clear job claim: clear-job-claim: failed to send RPC: sending to all replicas failed; last error: ba: Scan [/Table/15/1/880888731065417729,/Table/15/1/880888731065417730), [txn: 634b35a3], [can-forward-ts] RPC error: node unavailable; try another peer
W230709 09:55:34.660501 157161 jobs/adopt.go:518  [T1,n1] 6544  could not clear job claim: clear-job-claim: node unavailable; try another peer
W230709 09:55:34.660909 119866 jobs/adopt.go:518  [T1,n1] 6545  could not clear job claim: clear-job-claim: node unavailable; try another peer
I230709 09:55:34.685824 95803 kv/kvclient/kvcoord/transport_race.go:104  [-] 6546  transport race promotion: ran 444 iterations on up to 10980 requests
I230709 09:55:34.693032 95645 server/server_controller_orchestration.go:263  [T1,n1] 6547  server controller shutting down ungracefully
I230709 09:55:34.693179 95645 server/server_controller_orchestration.go:274  [T1,n1] 6548  waiting for tenant servers to report stopped
W230709 09:55:34.693864 95645 server/server_sql.go:1704  [T1,n1] 6549  server shutdown without a prior graceful drain
--- FAIL: TestTxnObeysTableModificationTime (236.31s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTxnObeysTableModificationTime.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-29587

sql/catalog/lease.TestTxnObeysTableModificationTime [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10846310?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10846310?buildTab=artifacts#/) on release-23.1 @ [f62527e83e458f5f4521497e7bfd5c60b70f1e31](https://github.com/cockroachdb/cockroach/commits/f62527e83e458f5f4521497e7bfd5c60b70f1e31):


```
E230710 09:51:29.153086 1483757 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r46/1:/Table/4{4-5}] 6575 +  | 	GOROOT/src/runtime/proc.go:6326
E230710 09:51:29.153086 1483757 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r46/1:/Table/4{4-5}] 6575 +  | runtime.doInit
E230710 09:51:29.153086 1483757 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r46/1:/Table/4{4-5}] 6575 +  | 	GOROOT/src/runtime/proc.go:6326
E230710 09:51:29.153086 1483757 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r46/1:/Table/4{4-5}] 6575 +  | runtime.main
E230710 09:51:29.153086 1483757 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r46/1:/Table/4{4-5}] 6575 +  | 	GOROOT/src/runtime/proc.go:233
E230710 09:51:29.153086 1483757 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r46/1:/Table/4{4-5}] 6575 +  | runtime.goexit
E230710 09:51:29.153086 1483757 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r46/1:/Table/4{4-5}] 6575 +  | 	src/runtime/asm_amd64.s:1594
E230710 09:51:29.153086 1483757 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r46/1:/Table/4{4-5}] 6575 +Wraps: (2) queue stopped
E230710 09:51:29.153086 1483757 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r46/1:/Table/4{4-5}] 6575 +Error types: (1) *withstack.withStack (2) *errutil.leafError
E230710 09:51:29.155151 1483659 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6577  unable to add: queue stopped
E230710 09:51:29.155151 1483659 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6577 +(1) attached stack trace
E230710 09:51:29.155151 1483659 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6577 +  -- stack trace:
E230710 09:51:29.155151 1483659 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6577 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver.init
E230710 09:51:29.155151 1483659 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6577 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/queue.go:221
E230710 09:51:29.155151 1483659 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6577 +  | runtime.doInit
E230710 09:51:29.155151 1483659 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6577 +  | 	GOROOT/src/runtime/proc.go:6349
E230710 09:51:29.155151 1483659 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6577 +  | runtime.doInit
E230710 09:51:29.155151 1483659 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6577 +  | 	GOROOT/src/runtime/proc.go:6326
E230710 09:51:29.155151 1483659 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6577 +  | runtime.doInit
E230710 09:51:29.155151 1483659 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6577 +  | 	GOROOT/src/runtime/proc.go:6326
E230710 09:51:29.155151 1483659 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6577 +  | runtime.main
E230710 09:51:29.155151 1483659 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6577 +  | 	GOROOT/src/runtime/proc.go:233
E230710 09:51:29.155151 1483659 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6577 +  | runtime.goexit
E230710 09:51:29.155151 1483659 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6577 +  | 	src/runtime/asm_amd64.s:1594
E230710 09:51:29.155151 1483659 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6577 +Wraps: (2) queue stopped
E230710 09:51:29.155151 1483659 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6577 +Error types: (1) *withstack.withStack (2) *errutil.leafError
E230710 09:51:29.154234 1483655 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r47/1:/Table/4{5-6}] 6576  unable to add: queue stopped
E230710 09:51:29.154234 1483655 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r47/1:/Table/4{5-6}] 6576 +(1) attached stack trace
E230710 09:51:29.154234 1483655 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r47/1:/Table/4{5-6}] 6576 +  -- stack trace:
E230710 09:51:29.154234 1483655 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r47/1:/Table/4{5-6}] 6576 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver.init
E230710 09:51:29.154234 1483655 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r47/1:/Table/4{5-6}] 6576 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/queue.go:221
E230710 09:51:29.154234 1483655 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r47/1:/Table/4{5-6}] 6576 +  | runtime.doInit
E230710 09:51:29.154234 1483655 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r47/1:/Table/4{5-6}] 6576 +  | 	GOROOT/src/runtime/proc.go:6349
E230710 09:51:29.154234 1483655 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r47/1:/Table/4{5-6}] 6576 +  | runtime.doInit
E230710 09:51:29.154234 1483655 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r47/1:/Table/4{5-6}] 6576 +  | 	GOROOT/src/runtime/proc.go:6326
E230710 09:51:29.154234 1483655 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r47/1:/Table/4{5-6}] 6576 +  | runtime.doInit
E230710 09:51:29.154234 1483655 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r47/1:/Table/4{5-6}] 6576 +  | 	GOROOT/src/runtime/proc.go:6326
E230710 09:51:29.154234 1483655 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r47/1:/Table/4{5-6}] 6576 +  | runtime.main
E230710 09:51:29.154234 1483655 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r47/1:/Table/4{5-6}] 6576 +  | 	GOROOT/src/runtime/proc.go:233
E230710 09:51:29.154234 1483655 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r47/1:/Table/4{5-6}] 6576 +  | runtime.goexit
E230710 09:51:29.154234 1483655 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r47/1:/Table/4{5-6}] 6576 +  | 	src/runtime/asm_amd64.s:1594
E230710 09:51:29.154234 1483655 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r47/1:/Table/4{5-6}] 6576 +Wraps: (2) queue stopped
E230710 09:51:29.154234 1483655 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r47/1:/Table/4{5-6}] 6576 +Error types: (1) *withstack.withStack (2) *errutil.leafError
E230710 09:51:29.166700 1476833 jobs/registry.go:989  [T1,n1] 6578  error getting live session: node unavailable; try another peer
W230710 09:51:29.170686 267836 sql/stats/automatic_stats.go:896  [T1,n1] 6579  failed to create statistics on table 58: create-stats: job 881171784782217217: node liveness error: restarting in background
W230710 09:51:29.860892 132971 ts/db.go:208  [T1,n1,ts-poll] 6580  error writing time series data: result is ambiguous: server shutdown
I230710 09:51:29.867335 94456 server/server_controller_orchestration.go:263  [T1,n1] 6581  server controller shutting down ungracefully
I230710 09:51:29.867597 94456 server/server_controller_orchestration.go:274  [T1,n1] 6582  waiting for tenant servers to report stopped
W230710 09:51:29.873249 94456 server/server_sql.go:1704  [T1,n1] 6583  server shutdown without a prior graceful drain
--- FAIL: TestTxnObeysTableModificationTime (238.41s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTxnObeysTableModificationTime.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/catalog/lease.TestTxnObeysTableModificationTime [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10861056?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10861056?buildTab=artifacts#/) on release-23.1 @ [a1062b43480d456afec57e6df469c87f9f9b0162](https://github.com/cockroachdb/cockroach/commits/a1062b43480d456afec57e6df469c87f9f9b0162):


```
E230711 10:05:26.794681 1484427 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r31/1:/{Table/29-NamespaceTab…}] 6579 +  | 	GOROOT/src/runtime/proc.go:6326
E230711 10:05:26.794681 1484427 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r31/1:/{Table/29-NamespaceTab…}] 6579 +  | runtime.doInit
E230711 10:05:26.794681 1484427 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r31/1:/{Table/29-NamespaceTab…}] 6579 +  | 	GOROOT/src/runtime/proc.go:6326
E230711 10:05:26.794681 1484427 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r31/1:/{Table/29-NamespaceTab…}] 6579 +  | runtime.main
E230711 10:05:26.794681 1484427 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r31/1:/{Table/29-NamespaceTab…}] 6579 +  | 	GOROOT/src/runtime/proc.go:233
E230711 10:05:26.794681 1484427 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r31/1:/{Table/29-NamespaceTab…}] 6579 +  | runtime.goexit
E230711 10:05:26.794681 1484427 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r31/1:/{Table/29-NamespaceTab…}] 6579 +  | 	src/runtime/asm_amd64.s:1594
E230711 10:05:26.794681 1484427 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r31/1:/{Table/29-NamespaceTab…}] 6579 +Wraps: (2) queue stopped
E230711 10:05:26.794681 1484427 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r31/1:/{Table/29-NamespaceTab…}] 6579 +Error types: (1) *withstack.withStack (2) *errutil.leafError
E230711 10:05:26.795243 1484556 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r50/1:/Table/{48-50}] 6580  unable to add: queue stopped
E230711 10:05:26.795243 1484556 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r50/1:/Table/{48-50}] 6580 +(1) attached stack trace
E230711 10:05:26.795243 1484556 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r50/1:/Table/{48-50}] 6580 +  -- stack trace:
E230711 10:05:26.795243 1484556 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r50/1:/Table/{48-50}] 6580 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver.init
E230711 10:05:26.795243 1484556 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r50/1:/Table/{48-50}] 6580 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/queue.go:221
E230711 10:05:26.795243 1484556 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r50/1:/Table/{48-50}] 6580 +  | runtime.doInit
E230711 10:05:26.795243 1484556 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r50/1:/Table/{48-50}] 6580 +  | 	GOROOT/src/runtime/proc.go:6349
E230711 10:05:26.795243 1484556 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r50/1:/Table/{48-50}] 6580 +  | runtime.doInit
E230711 10:05:26.795243 1484556 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r50/1:/Table/{48-50}] 6580 +  | 	GOROOT/src/runtime/proc.go:6326
E230711 10:05:26.795243 1484556 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r50/1:/Table/{48-50}] 6580 +  | runtime.doInit
E230711 10:05:26.795243 1484556 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r50/1:/Table/{48-50}] 6580 +  | 	GOROOT/src/runtime/proc.go:6326
E230711 10:05:26.795243 1484556 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r50/1:/Table/{48-50}] 6580 +  | runtime.main
E230711 10:05:26.795243 1484556 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r50/1:/Table/{48-50}] 6580 +  | 	GOROOT/src/runtime/proc.go:233
E230711 10:05:26.795243 1484556 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r50/1:/Table/{48-50}] 6580 +  | runtime.goexit
E230711 10:05:26.795243 1484556 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r50/1:/Table/{48-50}] 6580 +  | 	src/runtime/asm_amd64.s:1594
E230711 10:05:26.795243 1484556 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r50/1:/Table/{48-50}] 6580 +Wraps: (2) queue stopped
E230711 10:05:26.795243 1484556 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r50/1:/Table/{48-50}] 6580 +Error types: (1) *withstack.withStack (2) *errutil.leafError
E230711 10:05:26.795899 1484524 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r64/1:/Table/106/{2-4}] 6581  unable to add: queue stopped
E230711 10:05:26.795899 1484524 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r64/1:/Table/106/{2-4}] 6581 +(1) attached stack trace
E230711 10:05:26.795899 1484524 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r64/1:/Table/106/{2-4}] 6581 +  -- stack trace:
E230711 10:05:26.795899 1484524 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r64/1:/Table/106/{2-4}] 6581 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver.init
E230711 10:05:26.795899 1484524 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r64/1:/Table/106/{2-4}] 6581 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/queue.go:221
E230711 10:05:26.795899 1484524 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r64/1:/Table/106/{2-4}] 6581 +  | runtime.doInit
E230711 10:05:26.795899 1484524 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r64/1:/Table/106/{2-4}] 6581 +  | 	GOROOT/src/runtime/proc.go:6349
E230711 10:05:26.795899 1484524 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r64/1:/Table/106/{2-4}] 6581 +  | runtime.doInit
E230711 10:05:26.795899 1484524 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r64/1:/Table/106/{2-4}] 6581 +  | 	GOROOT/src/runtime/proc.go:6326
E230711 10:05:26.795899 1484524 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r64/1:/Table/106/{2-4}] 6581 +  | runtime.doInit
E230711 10:05:26.795899 1484524 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r64/1:/Table/106/{2-4}] 6581 +  | 	GOROOT/src/runtime/proc.go:6326
E230711 10:05:26.795899 1484524 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r64/1:/Table/106/{2-4}] 6581 +  | runtime.main
E230711 10:05:26.795899 1484524 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r64/1:/Table/106/{2-4}] 6581 +  | 	GOROOT/src/runtime/proc.go:233
E230711 10:05:26.795899 1484524 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r64/1:/Table/106/{2-4}] 6581 +  | runtime.goexit
E230711 10:05:26.795899 1484524 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r64/1:/Table/106/{2-4}] 6581 +  | 	src/runtime/asm_amd64.s:1594
E230711 10:05:26.795899 1484524 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r64/1:/Table/106/{2-4}] 6581 +Wraps: (2) queue stopped
E230711 10:05:26.795899 1484524 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r64/1:/Table/106/{2-4}] 6581 +Error types: (1) *withstack.withStack (2) *errutil.leafError
W230711 10:05:26.802923 269955 sql/stats/automatic_stats.go:896  [T1,n1] 6582  failed to create statistics on table 46: create-stats: error getting live session: node unavailable; try another peer
I230711 10:05:26.930698 96415 kv/kvclient/kvcoord/transport_race.go:104  [-] 6583  transport race promotion: ran 393 iterations on up to 10254 requests
W230711 10:05:27.453752 134731 ts/db.go:208  [T1,n1,ts-poll] 6584  error writing time series data: result is ambiguous: server shutdown
I230711 10:05:27.458881 96306 server/server_controller_orchestration.go:263  [T1,n1] 6585  server controller shutting down ungracefully
I230711 10:05:27.459049 96306 server/server_controller_orchestration.go:274  [T1,n1] 6586  waiting for tenant servers to report stopped
W230711 10:05:27.465002 96306 server/server_sql.go:1704  [T1,n1] 6587  server shutdown without a prior graceful drain
--- FAIL: TestTxnObeysTableModificationTime (238.53s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTxnObeysTableModificationTime.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/catalog/lease.TestTxnObeysTableModificationTime [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10861052?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10861052?buildTab=artifacts#/) on release-23.1 @ [a1062b43480d456afec57e6df469c87f9f9b0162](https://github.com/cockroachdb/cockroach/commits/a1062b43480d456afec57e6df469c87f9f9b0162):


```
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1686
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1621
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | github.com/cockroachdb/cockroach/pkg/sql/gcjob.persistProgress
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | 	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job_utils.go:281
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | github.com/cockroachdb/cockroach/pkg/sql/gcjob.schemaChangeGCResumer.deleteDataAndWaitForGC
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | 	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:333
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | github.com/cockroachdb/cockroach/pkg/sql/gcjob.schemaChangeGCResumer.Resume
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | 	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:314
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine.func2
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1628
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1629
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:474
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).resumeJob.func2
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:388
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | [...repeated from below...]
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +Wraps: (6) failed to read query result
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +Wraps: (7) candidate pg code: 57014
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +Wraps: (8) attached stack trace
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  -- stack trace:
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | github.com/cockroachdb/cockroach/pkg/util/cancelchecker.init
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | 	github.com/cockroachdb/cockroach/pkg/util/cancelchecker/cancel_checker.go:80
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | runtime.doInit
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | 	GOROOT/src/runtime/proc.go:6349
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | runtime.doInit
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | 	GOROOT/src/runtime/proc.go:6326
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | runtime.doInit
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | 	GOROOT/src/runtime/proc.go:6326
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | runtime.doInit
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | 	GOROOT/src/runtime/proc.go:6326
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | runtime.doInit
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | 	GOROOT/src/runtime/proc.go:6326
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | runtime.doInit
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | 	GOROOT/src/runtime/proc.go:6326
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | runtime.main
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | 	GOROOT/src/runtime/proc.go:233
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | runtime.goexit
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +  | 	GOROOT/src/runtime/asm_amd64.s:1594
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +Wraps: (9) query execution canceled
W230711 10:29:13.158311 93619 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881462322935955457] 5426 +Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *withstack.withStack (6) *errutil.withPrefix (7) *pgerror.withCandidateCode (8) *withstack.withStack (9) *errutil.leafError
E230711 10:29:13.159155 93619 jobs/registry.go:989  [T1,n1] 5427  error getting live session: node unavailable; try another peer
I230711 10:29:13.157745 81799 rpc/context.go:2318  [T1,n1,rnode=1,raddr=127.0.0.1:41589,class=default,rpc] 5422  connection heartbeat loop ended with err: <nil>
W230711 10:29:13.159889 89068 jobs/adopt.go:518  [T1,n1] 5428  could not clear job claim: clear-job-claim: failed to send RPC: sending to all replicas failed; last error: ba: Scan [/Table/15/1/102,/Table/15/1/103), [txn: c898ca8b], [can-forward-ts] RPC error: node unavailable; try another peer
W230711 10:29:13.165257 83385 jobs/adopt.go:518  [T1,n1] 5429  could not clear job claim: clear-job-claim: node unavailable; try another peer
I230711 10:29:13.165950 81223 server/server_controller_orchestration.go:263  [T1,n1] 5430  server controller shutting down ungracefully
I230711 10:29:13.165982 81223 server/server_controller_orchestration.go:274  [T1,n1] 5431  waiting for tenant servers to report stopped
W230711 10:29:13.166044 81223 server/server_sql.go:1704  [T1,n1] 5432  server shutdown without a prior graceful drain
--- FAIL: TestTxnObeysTableModificationTime (1.64s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTxnObeysTableModificationTime.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

just dump some early observation here:
1. all the failures in this ticket (so far) are all `stress` related.
2. the test was fine if I limited the CPU of stressing.
3. I was able to repro the panic simply with `./dev test pkg/sql/catalog/lease --filter 'TestTxnObeysTableModificationTime' --stress` cpu not limited, and it failed with the attached trace:
```
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240  failed to update job's progress payload or running status err: job 881594811748581377: select-job: failed to read query result: query execution canceled
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +(1) attached stack trace
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  -- stack trace:
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.update.func2
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:83
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.update
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:108
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.Update
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:378
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.SetProgress
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/jobs.go:680
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql/gcjob.persistProgress.func1
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job_utils.go:286
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn.func1
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1620
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn.func4
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1698
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/kv.runTxn.func1
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:965
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/kv/txn.go:928
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/kv.runTxn
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:964
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:927
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:902
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1686
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1621
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql/gcjob.persistProgress
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job_utils.go:281
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql/gcjob.schemaChangeGCResumer.deleteDataAndWaitForGC
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:333
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql/gcjob.schemaChangeGCResumer.Resume
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:314
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine.func2
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1628
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1629
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:474
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).resumeJob.func2
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:388
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +Wraps: (2) job 881594811748581377
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +Wraps: (3) attached stack trace
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  -- stack trace:
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).execInternal.func1.1
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:976
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next.func1
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:425
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:432
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).queryInternalBuffered
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:590
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).QueryRowExWithCols
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:642
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).QueryRowEx
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:628
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.update
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:102
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.Update
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:378
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.SetProgress
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/jobs.go:680
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql/gcjob.persistProgress.func1
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job_utils.go:286
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn.func1
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1620
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn.func4
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1698
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/kv.runTxn.func1
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:965
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/kv/txn.go:928
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/kv.runTxn
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:964
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:927
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:902
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1686
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1621
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql/gcjob.persistProgress
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job_utils.go:281
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql/gcjob.schemaChangeGCResumer.deleteDataAndWaitForGC
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:333
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql/gcjob.schemaChangeGCResumer.Resume
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:314
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine.func2
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1628
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1629
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:474
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).resumeJob.func2
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:388
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +Wraps: (4) select-job
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +Wraps: (5) attached stack trace
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  -- stack trace:
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*ieResultChannel).firstResult
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal_result_channel.go:125
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).execInternal
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1104
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).queryInternalBuffered
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:584
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).QueryRowExWithCols
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:642
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).QueryRowEx
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:628
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.update
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:102
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.Update
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:378
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.SetProgress
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/jobs.go:680
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql/gcjob.persistProgress.func1
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job_utils.go:286
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn.func1
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1620
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn.func4
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1698
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/kv.runTxn.func1
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:965
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/kv/txn.go:928
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/kv.runTxn
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:964
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:927
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:902
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1686
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1621
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql/gcjob.persistProgress
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job_utils.go:281
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql/gcjob.schemaChangeGCResumer.deleteDataAndWaitForGC
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:333
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/sql/gcjob.schemaChangeGCResumer.Resume
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:314
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine.func2
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1628
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1629
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:474
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).resumeJob.func2
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:388
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | [...repeated from below...]
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +Wraps: (6) failed to read query result
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +Wraps: (7) candidate pg code: 57014
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +Wraps: (8) attached stack trace
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  -- stack trace:
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | github.com/cockroachdb/cockroach/pkg/util/cancelchecker.init
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	github.com/cockroachdb/cockroach/pkg/util/cancelchecker/cancel_checker.go:80
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | runtime.doInit
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	GOROOT/src/runtime/proc.go:6349
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | runtime.doInit
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	GOROOT/src/runtime/proc.go:6326
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | runtime.doInit
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	GOROOT/src/runtime/proc.go:6326
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | runtime.doInit
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	GOROOT/src/runtime/proc.go:6326
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | runtime.doInit
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	GOROOT/src/runtime/proc.go:6326
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | runtime.doInit
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	GOROOT/src/runtime/proc.go:6326
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | runtime.main
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	GOROOT/src/runtime/proc.go:233
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | runtime.goexit
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +  | 	GOROOT/src/runtime/asm_amd64.s:1594
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +Wraps: (9) query execution canceled
W230711 21:43:05.671776 18850 sql/gcjob/gc_job_utils.go:299  [T1,n1,job=SCHEMA CHANGE GC id=881594811748581377] 240 +Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *withstack.withStack (6) *errutil.withPrefix (7) *pgerror.withCandidateCode (8) *withstack.withStack (9) *errutil.leafError
W230711 21:43:05.669582 19535 1@sql/event_log.go:659  [T1,n1,client=127.0.0.1:53974,hostssl,user=root] 238  unable to save 1 entries to system.eventlog: log-event: failed to read query result: query execution canceled
W230711 21:43:05.680601 11200 jobs/adopt.go:518  [T1,n1] 244  could not clear job claim: clear-job-claim: failed to send RPC: sending to all replicas failed; last error: ba: Scan [/Table/15/1/102,/Table/15/1/103), [txn: 9ea15958], [can-forward-ts] RPC error: node unavailable; try another peer
I230711 21:43:05.692385 16 server/server_controller_orchestration.go:263  [T1,n1] 245  server controller shutting down ungracefully
I230711 21:43:05.692439 16 server/server_controller_orchestration.go:274  [T1,n1] 246  waiting for tenant servers to report stopped
W230711 21:43:05.692525 16 server/server_sql.go:1704  [T1,n1] 247  server shutdown without a prior graceful drain
--- FAIL: TestTxnObeysTableModificationTime (3.73s)
    lease_test.go:1098: expected deadline exceeded, got: pq: restart transaction: liveness session expired 905.881036ms before transaction
FAIL
```
The main error is on liveness:
```
--- FAIL: TestTxnObeysTableModificationTime (3.73s)
    lease_test.go:1098: expected deadline exceeded, got: pq: restart transaction: liveness session expired 905.881036ms before transaction
FAIL
```

sql/catalog/lease.TestTxnObeysTableModificationTime [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10878670?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10878670?buildTab=artifacts#/) on release-23.1 @ [372c3aec05e32121be41f3b2866a6ba9c96097c3](https://github.com/cockroachdb/cockroach/commits/372c3aec05e32121be41f3b2866a6ba9c96097c3):


```
E230712 09:36:30.805307 1543014 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6538 +  | runtime.goexit
E230712 09:36:30.805307 1543014 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6538 +  | 	src/runtime/asm_amd64.s:1594
E230712 09:36:30.805307 1543014 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6538 +Wraps: (2) queue stopped
E230712 09:36:30.805307 1543014 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r53/1:/Table/5{2-3}] 6538 +Error types: (1) *withstack.withStack (2) *errutil.leafError
I230712 09:36:30.805808 95511 kv/kvserver/queue.go:614  [T1,n1,s1,r10/1:/Table/{6-7},raft] 6539  rate limited in MaybeAdd (merge): node unavailable; try another peer
E230712 09:36:30.806405 1543009 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r28/1:/Table/2{6-7}] 6540  unable to add: queue stopped
E230712 09:36:30.806405 1543009 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r28/1:/Table/2{6-7}] 6540 +(1) attached stack trace
E230712 09:36:30.806405 1543009 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r28/1:/Table/2{6-7}] 6540 +  -- stack trace:
E230712 09:36:30.806405 1543009 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r28/1:/Table/2{6-7}] 6540 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver.init
E230712 09:36:30.806405 1543009 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r28/1:/Table/2{6-7}] 6540 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/queue.go:221
E230712 09:36:30.806405 1543009 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r28/1:/Table/2{6-7}] 6540 +  | runtime.doInit
E230712 09:36:30.806405 1543009 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r28/1:/Table/2{6-7}] 6540 +  | 	GOROOT/src/runtime/proc.go:6349
E230712 09:36:30.806405 1543009 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r28/1:/Table/2{6-7}] 6540 +  | runtime.doInit
E230712 09:36:30.806405 1543009 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r28/1:/Table/2{6-7}] 6540 +  | 	GOROOT/src/runtime/proc.go:6326
E230712 09:36:30.806405 1543009 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r28/1:/Table/2{6-7}] 6540 +  | runtime.doInit
E230712 09:36:30.806405 1543009 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r28/1:/Table/2{6-7}] 6540 +  | 	GOROOT/src/runtime/proc.go:6326
E230712 09:36:30.806405 1543009 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r28/1:/Table/2{6-7}] 6540 +  | runtime.main
E230712 09:36:30.806405 1543009 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r28/1:/Table/2{6-7}] 6540 +  | 	GOROOT/src/runtime/proc.go:233
E230712 09:36:30.806405 1543009 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r28/1:/Table/2{6-7}] 6540 +  | runtime.goexit
E230712 09:36:30.806405 1543009 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r28/1:/Table/2{6-7}] 6540 +  | 	src/runtime/asm_amd64.s:1594
E230712 09:36:30.806405 1543009 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r28/1:/Table/2{6-7}] 6540 +Wraps: (2) queue stopped
E230712 09:36:30.806405 1543009 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r28/1:/Table/2{6-7}] 6540 +Error types: (1) *withstack.withStack (2) *errutil.leafError
I230712 09:36:30.807384 95443 server/start_listen.go:103  [T1,n1] 6542  server shutting down: instructing cmux to stop accepting
E230712 09:36:30.806919 1543034 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r29/1:/Table/2{7-8}] 6541  unable to add: queue stopped
E230712 09:36:30.806919 1543034 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r29/1:/Table/2{7-8}] 6541 +(1) attached stack trace
E230712 09:36:30.806919 1543034 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r29/1:/Table/2{7-8}] 6541 +  -- stack trace:
E230712 09:36:30.806919 1543034 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r29/1:/Table/2{7-8}] 6541 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver.init
E230712 09:36:30.806919 1543034 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r29/1:/Table/2{7-8}] 6541 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/queue.go:221
E230712 09:36:30.806919 1543034 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r29/1:/Table/2{7-8}] 6541 +  | runtime.doInit
E230712 09:36:30.806919 1543034 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r29/1:/Table/2{7-8}] 6541 +  | 	GOROOT/src/runtime/proc.go:6349
E230712 09:36:30.806919 1543034 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r29/1:/Table/2{7-8}] 6541 +  | runtime.doInit
E230712 09:36:30.806919 1543034 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r29/1:/Table/2{7-8}] 6541 +  | 	GOROOT/src/runtime/proc.go:6326
E230712 09:36:30.806919 1543034 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r29/1:/Table/2{7-8}] 6541 +  | runtime.doInit
E230712 09:36:30.806919 1543034 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r29/1:/Table/2{7-8}] 6541 +  | 	GOROOT/src/runtime/proc.go:6326
E230712 09:36:30.806919 1543034 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r29/1:/Table/2{7-8}] 6541 +  | runtime.main
E230712 09:36:30.806919 1543034 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r29/1:/Table/2{7-8}] 6541 +  | 	GOROOT/src/runtime/proc.go:233
E230712 09:36:30.806919 1543034 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r29/1:/Table/2{7-8}] 6541 +  | runtime.goexit
E230712 09:36:30.806919 1543034 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r29/1:/Table/2{7-8}] 6541 +  | 	src/runtime/asm_amd64.s:1594
E230712 09:36:30.806919 1543034 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r29/1:/Table/2{7-8}] 6541 +Wraps: (2) queue stopped
E230712 09:36:30.806919 1543034 kv/kvserver/queue.go:706  [T1,n1,consistencyChecker,s1,r29/1:/Table/2{7-8}] 6541 +Error types: (1) *withstack.withStack (2) *errutil.leafError
I230712 09:36:30.809435 95728 rpc/context.go:2318  [T1,n1,rnode=1,raddr=127.0.0.1:38095,class=default,rpc] 6543  connection heartbeat loop ended with err: <nil>
I230712 09:36:30.813829 98050 jobs/registry.go:1588  [T1,n1] 6544  AUTO SPAN CONFIG RECONCILIATION job 881734305623965697: stepping through state succeeded
W230712 09:36:30.823280 281160 jobs/adopt.go:518  [T1,n1] 6545  could not clear job claim: clear-job-claim: failed to send RPC: sending to all replicas failed; last error: ba: Scan [/Table/15/1/101,/Table/15/1/102), [txn: f5926721], [can-forward-ts] RPC error: node unavailable; try another peer
W230712 09:36:30.827572 281053 jobs/adopt.go:518  [T1,n1] 6546  could not clear job claim: clear-job-claim: failed to send RPC: sending to all replicas failed; last error: ba: Scan [/Table/15/1/100,/Table/15/1/101), [txn: 1c775c9f], [can-forward-ts] RPC error: node unavailable; try another peer
W230712 09:36:30.828429 118559 jobs/adopt.go:518  [T1,n1] 6547  could not clear job claim: clear-job-claim: node unavailable; try another peer
I230712 09:36:31.067711 95409 kv/kvclient/kvcoord/transport_race.go:104  [-] 6548  transport race promotion: ran 454 iterations on up to 11025 requests
I230712 09:36:31.071389 95314 server/server_controller_orchestration.go:263  [T1,n1] 6549  server controller shutting down ungracefully
I230712 09:36:31.071554 95314 server/server_controller_orchestration.go:274  [T1,n1] 6550  waiting for tenant servers to report stopped
W230712 09:36:31.072234 95314 server/server_sql.go:1704  [T1,n1] 6551  server shutdown without a prior graceful drain
--- FAIL: TestTxnObeysTableModificationTime (236.19s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTxnObeysTableModificationTime.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/sql/catalog/lease/lease_test.go
+++ b/pkg/sql/catalog/lease/lease_test.go
@@ -1004,7 +1004,7 @@ func TestTxnObeysTableModificationTime(t *testing.T) {
 	//
 	// In order to mitigate that push, we increase the target_duration when the
 	// test is run under race.
-	if util.RaceEnabled {
+	if util.RaceEnabled || skip.Stress() {
 		_, err := sqlDB.Exec(
 			"SET CLUSTER SETTING kv.closed_timestamp.target_duration = '120s'")
 		require.NoError(t, err)
@@ -1181,6 +1181,14 @@ INSERT INTO t.kv VALUES ('a', 'b');
 	// index) and clear the index keys.
 	if _, err := sqltestutils.AddImmediateGCZoneConfig(sqlDB, tableDesc.GetID()); err != nil {
 		t.Fatal(err)
+	}
+
+	// Reset closed timestamp so that deleted keys can be GC'ed within the
+	// SucceedSoon window.
+	if util.RaceEnabled || skip.Stress() {
+		_, err := sqlDB.Exec(
+			"SET CLUSTER SETTING kv.closed_timestamp.target_duration = '3s'")
+		require.NoError(t, err)
 	}
 
 	testutils.SucceedsSoon(t, func() error {

